src/frontend/views/mixins/breadcrumb.pug

```diff
@@ -1,6 +1,12 @@
-mixin breadcrumb(elements)
+mixin breadcrumb(resource, record, actionName)
   nav.breadcrumb(aria-label="breadcrumbs")
     ul
-      for element in elements
-        li(class={ 'is-active' : elements.indexOf(element) === elements.length - 1 })
-          a(href=h.listUrl(currentResource))=element
```

Comment:
previously helpers always directed to list view (even when I clicked record title). Fixed that